src/capi/setup.py

```diff
@@ -17,7 +17,7 @@
 
 # Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
-VERSION = '0.0.6'
+VERSION = '0.0.7'
```

Comment:
Should we make a v0.1.0 release?

We are starting to take more dependencies on this project for SIG-Windows testing in Kubernetes

Reply:
Sure, that would make it possible to do start doing proper patch releases. Maybe we can do the next release as v0.0.7 and then the next is v0.1.0?

Reply:
Scratch that, I'll update this so the next release is v0.1.0.